packages/dbus/build.sh

```diff
@@ -2,29 +2,25 @@ TERMUX_PKG_HOMEPAGE=https://dbus.freedesktop.org
 TERMUX_PKG_DESCRIPTION="Freedesktop.org message bus system"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.15.6
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION="1.16.2"
 TERMUX_PKG_SRCURL="https://dbus.freedesktop.org/releases/dbus/dbus-$TERMUX_PKG_VERSION.tar.xz"
-TERMUX_PKG_SHA256=f97f5845f9c4a5a1fb3df67dfa9e16b5a3fd545d348d6dc850cb7ccc9942bd8c
+TERMUX_PKG_SHA256=0ba2a1a4b16afe7bceb2c07e9ce99a8c2c3508e5dec290dbb643384bd6beb7e2
 TERMUX_PKG_DEPENDS="libexpat, libx11"
 TERMUX_PKG_BREAKS="dbus-dev"
 TERMUX_PKG_REPLACES="dbus-dev"
-TERMUX_PKG_BUILD_IN_SRC=true
 
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
-ac_cv_func_close_range=no
---enable-ld-version-script
---disable-libaudit
---disable-systemd
---disable-tests
---enable-x11-autolaunch
---with-test-socket-dir=$TERMUX_PREFIX/tmp
---with-session-socket-dir=$TERMUX_PREFIX/tmp
---with-x=auto
+-Dlibaudit=disabled
+-Dsystemd=disabled
+-Dmodular_tests=disabled
+-Dx11_autolaunch=enabled
+-Dtest_socket_dir=$TERMUX_PREFIX/tmp
+-Dsession_socket_dir=$TERMUX_PREFIX/tmp
 "
 
 termux_step_pre_configure() {
 	export LIBS="-llog"
+	rm CMakeLists.txt
```

Comment:
Why are we removing the CMakeLists file?
Repeatability?

I think a comment would help here.

Reply:
It is needed to enforce meson building.

Reply:
Should there be a `TERMUX_PKG_FORCE_MESON` variable?

Reply:
There is no `TERMUX_PKG_FORCE_MESON` in termux (AFAIK).